pages/partials/mathjax.ejs

```diff
@@ -0,0 +1,20 @@
+<script>
+ var MathJax = {
+     tex: {
+         inlineMath: [['$', '$'], ['\\(', '\\)']]
+     },
+     svg: {
+         fontCache: 'global'
+     },
+     loader: {
+         load: ['input/tex-base', 'output/svg']
+     },
+     startup: {
+         ready: () => {
+             MathJax.startup.defaultReady();
+             MathJax.Hub = { Queue: function(){ MathJax.typesetPromise(); } };
```

Comment:
If I'm understanding correctly, this is for backwards compatibility so that our existing code can continue to call `MathJax.Hub.Queue` to trigger typesetting? Very nice! But maybe we should explicitly convert the few direct callers in PL itself? I think these are:
```
elements/pl-symbolic-input/pl-symbolic-input.mustache
public/localscripts/calculationQuestion/SimpleClient.js
testCourse/courseInstances/Sp15/assessments/exam1-automaticTestSuite/clientFilesAssessment/formulas.html
```